tests/specs/query.js

```diff
@@ -61,26 +61,12 @@
             };
         });
 
-        afterEach(function (done) {
+        afterEach(function () {
             if (this.server && !this.server.isClosed()) {
                 this.server.close();
             }
 
-            var spec = this;
-
             var req = indexedDB.deleteDatabase(this.dbName);
-
-            req.onsuccess = function () {
-                done();
-            };
-
-            req.onerror = function () {
-                console.log('failed to delete db in afterEach', arguments, spec);
-            };
-
-            req.onblocked = function () {
-                console.log('db blocked', arguments);
-            };
         });

```

Comment:
Is there a reason you removed `done` in this block here? I think `deleteDatabase` might not always immediately complete its action which could be a problem if the next test depends on the database being deleted.